src/components/spaces/cell-zoom/organism-container.tsx

```diff
@@ -0,0 +1,58 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "../../base";
+
+import "./organism-view.sass";
+import { ZoomControl } from "../../zoom-control";
+import { OrganismView } from "./organism-view";
+
+import { CellZoomComponent } from "../cell-zoom/cell-zoom";
+
+interface IProps extends IBaseProps {
+  rowIndex: number;
+}
+interface IState {
+  zoomLevel: number;
```

Comment:
@dstrawberrygirl Do you this is would work to integrate this with the `CellZoomRowModel.zoomLevel` property?